usaspending_api/search/v2/views/search.py

```diff
@@ -70,3 +78,273 @@ def post(self, request):
         response['results'] = results
 
         return Response(response)
+
+
+class SpendingByCategoryVisualizationViewSet(APIView):
+
+    def post(self, request):
+        """Return all budget function/subfunction titles matching the provided search text"""
+        json_request = request.data
+        category = json_request.get('category', None)
+        scope = json_request.get('scope', None)
+        filters = json_request.get('filters', None)
+        limit = json_request.get('limit', None)
+        page = json_request.get('page', None)
+
+        if category is None:
+            raise InvalidParameterException('Missing one or more required request parameters: category')
+        potential_categories = ["awarding_agency", "funding_agency", "recipient", "cfda_programs", "industry_codes"]
+        if category not in potential_categories:
+            raise InvalidParameterException('Category does not have a valid value')
+        if (scope is None) and (category != "cfda_programs"):
+            raise InvalidParameterException('Missing one or more required request parameters: scope')
+        if limit is None:
```

Comment:
Instead of doing an if statement here, why not change the initialization of `limit` above to be `limit = json_request.get('limit', 10)`?